suites/luminous/ansible/sanity_ceph_ansible.yaml

```diff
@@ -675,31 +675,6 @@ tests:
       polarion-id: CEPH-9581
       module: test_ansible_roll_over.py
       config:
-          ansi_config:
-              ceph_test: True
-              ceph_origin: distro
-              ceph_stable_release: luminous
-              ceph_repository: rhcs
-              osd_scenario: collocated
-              osd_auto_discovery: False
-              journal_size: 1024
-              ceph_stable: True
-              ceph_stable_rh_storage: True
-              fetch_directory: ~/fetch
-              copy_admin_key: true
-              ceph_conf_overrides:
-                  global:
-                    osd_pool_default_pg_num: 64
-                    osd_default_pool_size: 2
-                    osd_pool_default_pgp_num: 64
-                    mon_max_pg_per_osd: 1024
-                  mon:
-                    mon_allow_pool_delete: true
-              cephfs_pools:
-                - name: "cephfs_data"
-                  pgs: "8"
-                - name: "cephfs_metadata"
-                  pgs: "8"
```

Comment:
@Ameenasuhani you have removed these configs in the yaml. Was this intentional ?. 
The same for below too.